src/styles/main.global.scss

```diff
@@ -142,6 +141,14 @@ li em {
   overflow-y: auto;
 }
 
+.smartbanner-show {
```

Comment:
these should not be needed no?

Reply:
I guess this is the class the shows up the smartbanner

Reply:
this class being given to the element from Show() and Hide() methods in the smartbanner container

Reply:
Maybe have that be a prop to `Container`?

Reply:
Hmm.. I guess I am lost where this is used :S

Reply:
```
hide = () => {
    if (window.document && window.document.querySelector('html')) {
      (window.document.querySelector('html') as Element).classList.remove(
        'smartbanner-show'
      );
    }
  };

  show = () => {
    if (window.document && window.document.querySelector('html')) {
      (window.document.querySelector('html') as Element).classList.add(
        'smartbanner-show'
      );
    }
  };
```

Reply:
Right. The essence of this needs to be corrected. If we wish to have some margin to leave space for the smart banner, then we bought to add this to `App.tsx`. Changing the html tag in React seems like a solution that will create bugs or unsustainable.

Reply:
So you need me to make based on a state ?